### ISSUE TITLE
Documentation of variables

### ISSUE THREAD

**Issue by alxschwrz** [maxspahn/robot_mpcs#5]
While the repo is quite handy to design and create mpc solvers, it is a bit tricky to understand the variables solely from the naming. I think a little documentation and/or renaming of some variables would help a lot for easier usage.

**Comment by alxschwrz** [comment]
Sure, I will propose some changes for the examples I am working with.

**Comment by alxschwrz** [comment]
You can find the first update in [ft-variable-naming](https://github.com/alxschwrz/robot_mpcs/tree/ft-variable-naming). Should I open a PR already?

**Comment by alxschwrz** [comment]
@maxspahn

**Comment by maxspahn** [comment]
yeah, sure. Then we can communicate about the naming directly in the PR.
